pages/thc/[processRef]/review.tsx

```diff
@@ -208,23 +154,19 @@ const ReviewPage: NextPage = () => {
         the answers with all present tenants and ask them to sign.
       </Paragraph>
       {tenantsPresent.length > 0 && (
-        <Paragraph>Present for check: {tenantsPresent.join(", ")}</Paragraph>
+        <Paragraph>
+          Present for check: {presentTenantNames.join(", ")}
+        </Paragraph>
       )}
-      {sections
-        .filter((section) => section.rows.length)
-        .map(({ heading, rows }) => (
-          <React.Fragment key={heading}>
-            <Heading level={HeadingLevels.H2}>{heading}</Heading>
-
-            {rows.length ? (
-              <SummaryList
-                rows={(rows as unknown) as { key: string; value: string }[]}
-              />
-            ) : (
-              <Paragraph>Loading...</Paragraph>
-            )}
-          </React.Fragment>
-        ))}
+      {tenantsPresent.map(({ fullName, id }) => (
+        <React.Fragment key={id}>
+          <Heading level={HeadingLevels.H2}>{fullName}</Heading>
+          <IdAndResidencyReviewSection selectedTenantId={id} />
+          <HouseholdReviewSection />
+          <PropertyInspectionReviewSection />
+          <WellbeingSupportReviewSection />
+        </React.Fragment>
+      ))}
```

Comment:
Ah I see what you've done. I think we probably want to only repeat the ID and residency check steps for each tenant, and to repeat it for all tenants, not just the present ones. We also ask for a signature from each present tenant, only. The housing officer would then start from the top of the page, confirming all of the details, get a tenant to sign, then repeat for the other present tenants.

If we do that, I think we want to pull the logic for iterating through the tenants into the `IdAndResidencyReviewSection`.